docs/semanticdb/specification.md

```diff
@@ -810,6 +811,12 @@ signatures in supported languages.
 
 `access`. [Access](#access) modifier of the corresponding definition.
 
+`documentation`. The documentation string associated with a symbol.
+
+The availability of this information depends on the underlying method used to
```

Comment:
This paragraph can be removed. All protobuf fields are optional and it's normal that different indexers emit different fields. It's the responsibility of each indexer to document what fields they emit.